recipe/meta.yaml

```diff
@@ -10,18 +10,16 @@ source:
   sha256: 84f9f229e4fc6c0df8a23620f2b03f277b1bfb15a497603c704743443e38d420
 
 build:
-  number: 2
+  number: 3
   noarch: python
 
 requirements:
   host:
     - python >=3.8
-    - dask-core {{ version }}
-    - distributed {{ version }}
   run:
     - python >=3.8
-    - {{ pin_compatible('dask-core', max_pin='x.x.x') }}
-    - {{ pin_compatible('distributed', max_pin='x.x.x') }}
+    - dask-core {{ version }}.*,!={{ version }}a.*
+    - distributed {{ version }}.*,!={{ version }}a.*
```

Comment:
My understanding here is that nightly versions get parsed by conda as something like `2022.8.1a.220805`, so the wildcard exclusion here should always be valid, I just think for some reason conda isn't a fan of exclusion wildcards without the `.`; either way, I've opened https://github.com/conda/conda/issues/11744 to document this behavior.